Makefile

```diff
@@ -3,10 +3,11 @@ VELA_VERSION ?= master
 # Repo info
 GIT_COMMIT          ?= git-$(shell git rev-parse --short HEAD)
 GIT_COMMIT_LONG     ?= $(shell git rev-parse HEAD)
-VELA_VERSION_VAR    := github.com/oam-dev/kubevela/version.VelaVersion
-VELA_GITVERSION_VAR := github.com/oam-dev/kubevela/version.GitRevision
-LDFLAGS             ?= "-X $(VELA_VERSION_VAR)=$(VELA_VERSION) -X $(VELA_GITVERSION_VAR)=$(GIT_COMMIT)"
+VELA_VERSION_KEY    := github.com/oam-dev/kubevela/version.VelaVersion
+VELA_GITVERSION_KEY := github.com/oam-dev/kubevela/version.GitRevision
+LDFLAGS             ?= "-s -w -X $(VELA_VERSION_KEY)=$(VELA_VERSION) -X $(VELA_GITVERSION_KEY)=$(GIT_COMMIT)"
```

Comment:
can you explain a bit more about `-s -w` flags? Or give some docs.

Reply:
Please refer to [go cmd link](https://golang.org/cmd/link)
```
-s
	Omit the symbol table and debug information.
-w
	Omit the DWARF symbol table.
```

Reply:
[Kubernetes Makefile](https://github.com/kubernetes/kubernetes/blob/master/build/root/Makefile)

```
#   make all GOLDFLAGS=""
#     Note: Specify GOLDFLAGS as an empty string for building unstripped binaries, which allows
#           you to use code debugging tools like delve. When GOLDFLAGS is unspecified, it defaults
#           to "-s -w" which strips debug information. Other flags that can be used for GOLDFLAGS 
#           are documented at https://golang.org/cmd/link/
```